src/components/Kanban.tsx

```diff
@@ -98,13 +178,18 @@ const Kanban = ({
   onQuery,
   resultKeys,
   parentUid,
+  views,
 }: {
   resultKeys: Column[];
   data: Result[];
   layout: Record<string, string | string[]>;
   onQuery: () => void;
   parentUid: string;
+  views: { column: string; mode: string; value: string }[];
 }) => {
+  const [localFoldStates, setLocalFoldStates] = useState(
+    new Map<string, boolean>()
+  );
```

Comment:
a lot of changes in this PR are from adding `localFoldStates`.

Problem this solves:
- user selects `embed` in `Kanban` view and chooses `embed` `open?` state
- user manually changes one or more block fold states
- user performs an action that causes a re-render
  - this includes hovering over the column menu or dragging and dropping cards
- manually changed blocks then revert to chosen `embed` state

This seems to happen less in `Table` view which leads me to believe this might be solved by investigating and addressing the root cause of the re-renders instead of adding `localFoldStates`, but I'm not certain.

Reply:
Pull out `CellEmbed` into its own component and see if you could live without the added state